waspc/waspls/src/Wasp/LSP/ExtImport/Path.hs

```diff
@@ -143,6 +158,10 @@ allowedExts DotTJSX = [".tsx", ".jsx"]
 allowedExts DotAnyTS = [".ts", ".js", ".tsx", ".jsx"]
 allowedExts (DotExact ext) = [ext]
 
+replaceExtension :: SP.Path' SP.Abs (SP.File a) -> String -> SP.Path' SP.Abs (SP.File a)
+replaceExtension path ext =
+  SP.fromPathAbsFile $ fromMaybe (error "invalid extension") $ P.replaceExtension ext $ SP.toPathAbsFile path
+
```

Comment:
This sounds like a pure StrongPath utility function, doesn't it? If so, let's move it to Wasp.Util.StrongPath.hs (create it if it doesn't exist yet). You could also open an issue for StrongPath to add this function to it, on StrongPath repo.